reactioncommerce/reaction-identity
# feat: update to Meteor 1.10.2

## Changes
Update to Meteor 1.10.2 with Node 12.16.1 and version 3.5.7 of the `mongodb` library. This resolves potential vulnerabilities in NPM dependencies.

## Testing
1. Run `make dev-reaction-identity` to test the development build (test login from storefront or admin).
2. Stop the identity service and delete the `docker-compose.override.yml` symlink.
3. Build production image with `docker build . -t reactioncommerce/identity:3.1.0`
4. Temporarily update the image in `docker-compose.yml` to be `reactioncommerce/identity:3.1.0`
5. Run `docker-compose up -d` and then verify that the identity service works (test login from storefront or admin).
